temporalio/activity.py

```diff
@@ -116,6 +116,7 @@ class Info:
     workflow_namespace: str
     workflow_run_id: str
     workflow_type: str
+    priority: temporalio.common.Priority
```

Comment:
Is priority always non-null in proto for activity task and workflow start event?

Reply:
No, it's not, but we decided that just means default. Not personally a huge fan of that, but, the information conveyed is the same https://github.com/temporalio/sdk-java/pull/2453#discussion_r2010311967

Reply:
Works for me. When it comes to us passing it in, does server care whether we send a null priority or a non-null one with default values? The reason I ask about that direction is because if we want always output to be non-`Optional`, we might as well never accept `Optional` input and just default to a fixed immutable `Priority.default`. So there's no such thing as `Optional[Priority]` anywhere (kinda touched on in another comment).